### ISSUE TITLE
ModuleNotFoundError: No module named 'torch_geometric'

### ISSUE THREAD

**Issue by galbc** [rusty1s/pytorch_geometric#1900]
## Environment

* OS:Mac OSX 10.14.6 (x86_64)
* Python version:3.7 (64-bit runtime)
* PyTorch version:1.7.0
* CUDA/cuDNN version:No CUDA
* GCC version:Could not collect
* How did you try to install PyTorch Geometric and its extensions (wheel, source):source
* Any other relevant information: 

## Checklist

- [ X] I followed the [installation guide](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html).
- [ X] I cannot find my error message in the [FAQ](https://pytorch-geometric.readthedocs.io/en/latest/notes/installation.html#id1).
- [ ] I set up CUDA correctly and can compile CUDA code via `nvcc`.
- [ ] I do have multiple CUDA versions on my machine.

## Additional context
I have created a conda environment with python 3.7.9, and torch is properly installed. Followed the instruction used the following commands to install torch-geometric:
$ pip3 install --upgared torch-scatter
$ pip3 install --upgared torch-sparse
$ pip3 install --upgared torch-cluster
$ pip3 install --upgared torch-spine-conv
$ pip3 install torch-geometric

However, torch-geometric and all needed packages can not be found in site-packages folder of the conda environment. And when tryingt to import torch_geometric I get this error: 
>>> import torch_geometric
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'torch_geometric'

**Comment by aGIToz** [comment]
You need use the conda packages of torch geometric.  pip generally will not install in conda virtural env.  Have a look at conda packages of torch_geometric [here](https://anaconda.org/search?q=torch-geometric).

**Comment by galbc** [comment]
> You need use the conda packages of torch geometric. pip generally will not install in conda virtural env. Have a look at conda packages of torch_geometric [here](https://anaconda.org/search?q=torch-geometric).

**Comment by galbc** [comment]
> conda packages of torch geometric
Thanks for your response. 
I uninstalled all torch geometric packages that I initially installed with pip3. but, I am not able to install the packages with conda.

**Comment by galbc** [comment]
@aGIToz 
Thanks for your response.
I uninstalled all torch geometric packages that I initially installed with pip3. but, I am not able to install the packages with conda.

**Comment by rusty1s** [comment]
Didn't know there exists already so many `torch-geometric` conda packages, awesome! In general, you can use `pip` to install into a conda environment. You just need to make sure that `pip` is linked to miniconda, in your case:
```
which pip3
```
should link to `pip` installed by miniconda.

**Comment by galbc** [comment]
Thanks a lot @rusty1s for your response. It is a very good point that pip should be linked to miniconda. This is exactly the issue I am struggling with. 

To resolve this issue, I created a new miniconda with python 3.8.5, the anaconda python version. I tried installing pip in the new miniconda environment (in: /Users/myName/opt/anaconda3/envs/envName) and used: conda install -c anaconda pip; however, it is still not linked to the miniconda and I get the following:
$ which pip
/Library/Frameworks/Python.framework/Versions/3.9/bin/pip
$ which pip3 
/Library/Frameworks/Python.framework/Versions/3.7/bin/pip3

Then I tried to unset the $PYTHONPATH and set it to "/Users/myName/opt/anaconda3/envs/envName" (not sure if it was a good idea). Anyhow, I faced the same problem. 

Finally, I decided to uninstall both python3.9 and 3.7 (decluttered my computer ;D).  So, now I only have python 3.8.5, the anaconda default python version which the miniconda was also created with this version. Now pip and pip3 are linked to this miniconda. And I successfully installed torch-geometric packages with pip3. 

Thanks
